### PR TITLE
Add 3D camera controls to publisher

### PR DESCRIPTION
**Pull request by mikkomustikkamaa**
PR contains changes to handle 3D camera controls in publisher.

### CODE REVIEW

**Comment by ZakarFin** [comment]
Does this work with editing an existing embedded map? I think CameraControls3dTool needs at least init method to detect if the tool should be on or not when initializing the publisher for editing.

**Comment by ZakarFin** [comment]
Otherwise looks ok for merging.

File: bundles/mapping/camera-controls-3d/tool/CameraControls3dTool.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    *
+    * @returns {Object} tool value object
+    */
+        getValues: function () {
+            var me = this;
+
+            if (me.state.enabled) {
+                return {
+                    configuration: {
+                        mapfull: {
```
Comment (ZakarFin) [inline bundles/mapping/camera-controls-3d/tool/CameraControls3dTool.js]: This is not part of mapfull plugins but a bundle called "camera-controls-3d". So getValues() should return something different. The featuredata bundle works in the same way so you could look for an example here https://github.com/oskariorg/oskari-frontend/blob/develop/bundles/framework/publisher2/tools/FeaturedataTool.js#L62

Comment (ZakarFin) [reply bundles/mapping/camera-controls-3d/tool/CameraControls3dTool.js]: You will also need to whitelist the bundle for publishing in oskari-server: https://github.com/oskariorg/oskari-server/blob/master/control-base/src/main/java/fi/nls/oskari/control/view/AppSetupHandler.java#L149

Comment (mikkomustikkamaa) [reply bundles/mapping/camera-controls-3d/tool/CameraControls3dTool.js]: Thank you for the tips. Camera controls are now handled as a bundle. Also https://github.com/oskariorg/oskari-server/pull/497 is created to whitelist the bundle.

**Pull request merged**